tests/Predis/Command/Redis/GEOSEARCH_Test.php

```diff
@@ -186,7 +186,7 @@ public function responsesProvider(): array
                 [
                     'member1' => ['lng' => 1.1, 'lat' => 2.2],
                     'member2' => ['lng' => 2.2, 'lat' => 3.3],
-                    'member3' => ['lng' => 3.3, 'lat' => 4.4]],
+                    'member3' => ['lng' => 3.3, 'lat' => 4.4], ],
```

Comment:
My hands are itching to move this bracket to a new line:

```suggestion
                    'member3' => ['lng' => 3.3, 'lat' => 4.4],
                ],
```